src/Trading/components/TradingPrice.tsx

```diff
@@ -64,7 +64,6 @@ const TradingPrice = React.forwardRef(function TradingPrice(props: TradingPriceP
       onChange={props.onChange}
       onFocus={props.selectOnFocus ? event => event.target.select() : undefined}
       style={props.style}
-      type="number"
```

Comment:
@ebma Do we still get to see the numpad rather than the default keyboard? 😉

Reply:
Yes, at least on iOS (didn't check on Android). I guess it's related to the pattern `pattern: "[0-9]*",` and not necessarily the `type="number"`.